### PR TITLE
Add public key conversion to JWK format.

### PR DESCRIPTION
**Pull request by mdetemad**
Fixes #issue

> It's a good idea to open an issue first for discussion.

- [x] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [x] `pom.xml` parent set to latest `shared-configuration`
- [ ] Appropriate changes to README are included in PR
- [ ] API's need to be enabled to test (tell us)
- [ ] Environment Variables need to be set (ask us to set them)
- [ ] **Tests** pass:   `mvn clean verify` **required**
- [ ] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [x] Please **merge** this PR for me once it is approved.

### CODE REVIEW

**Comment by mdetemad** [comment]
Added public key conversion to JWK format.

File: kms/pom.xml [hunk 1]
```diff
@@ -64,6 +63,16 @@
       <version>1.1.3</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+        <groupId>com.nimbusds</groupId>
+        <artifactId>nimbus-jose-jwt</artifactId>
+        <version>9.31</version>
+    </dependency>
+    <dependency>
```
Comment (averikitsch) [inline kms/pom.xml:71]: It doesn't look like this dependency is being used. Please remove.

File: kms/src/main/java/kms/ConvertPublicKeyToJwk.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+ * limitations under the License.
+ */
+
+ package kms;
+
+ // [START kms_get_public_key]
+ import com.google.cloud.kms.v1.CryptoKeyVersionName;
+ import com.google.cloud.kms.v1.KeyManagementServiceClient;
+ import com.google.cloud.kms.v1.PublicKey;
+ // NOTE: The library nimbusds is NOT endoresed for anything beyond conversion to JWK.
```
Comment (averikitsch) [inline kms/src/main/java/kms/ConvertPublicKeyToJwk.java:23]:
```suggestion
 // NOTE: The library nimbusds is NOT endorsed for anything beyond conversion to JWK.
```

**Review by averikitsch — CHANGES_REQUESTED** [review]

File: kms/pom.xml (hunk 1 above)
Comment (mdetemad) [reply kms/pom.xml:71]: Hi, thanks for looking at this.
Actually,  "nimbus-jose-jwt" is a dependency being used in this update.
It is used in java-docs-samples/kms/src/main/java/kms/ConvertPublicKeyToJwk.java

Comment (averikitsch) [reply kms/pom.xml:71]: Right, I'm referring to the `org.bouncycastle:bcpkix-jdk15on` dependency L71-75

**Comment by averikitsch** [comment]
Please fix the lint errors. Thanks!

**Review by Sita04 — APPROVED** [review]

**Comment by Sita04** [comment]
Kindly address Averi's comment before merging.

File: kms/pom.xml (hunk 1 above)
Comment (mdetemad) [reply kms/pom.xml:71]: That's also needed by the nimbus-jose-jwt.
When I remove that dependency, I see this error:
Screenshot 2023-04-20 at 9 42 29 AM

Comment (averikitsch) [reply kms/pom.xml:71]: Sorry about that. Thanks!

**Review by averikitsch — APPROVED** [review]

**Pull request merged**